pace-util/pace/util/caching_comm.py

```diff
@@ -49,12 +49,14 @@ class CachingCommData:
     size: int
     bcast_objects: List[Any] = dataclasses.field(default_factory=list)
     received_buffers: List[np.ndarray] = dataclasses.field(default_factory=list)
+    generic_obj_buffers: List[Any] = dataclasses.field(default_factory=list)
     split_data: List["CachingCommData"] = dataclasses.field(default_factory=list)
 
     def __post_init__(self):
         self._i_bcast = 0
         self._i_buffers = 0
         self._i_split = 0
+        self._i_generic_obj = 0
```

Comment:
"generic" has a specific meaning, I'd suggest `python_obj` or even just `obj` (all python classes inherit from `object` so this is accurate).

Reply:
what is this specific meaning that we are not upholding with the logic in this class?

Reply:
The types being stored are not specifically "generic types" (e.g. as discussed [here](https://docs.oracle.com/javase/tutorial/java/generics/types.html)).